### PR TITLE
JC-24. Create import

### PR DESCRIPTION
**Pull request by saffchen**

### CODE REVIEW

File: src/test/java/org/jacp/mapper/QuestionMapperTest.java [hunk 1]
```diff
@@ -24,29 +25,32 @@
 class QuestionMapperTest {
 
     @Mock
-    private QuestionEntity questionEntity;
+    private static QuestionEntity questionEntity;
```
Comment (pelmegov) [inline src/test/java/org/jacp/mapper/QuestionMapperTest.java:28]: Why static?

File: src/test/java/org/jacp/service/ImportServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    @InjectMocks
+    private ImportService importService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    static Long id1 = 1L;
```
Comment (pelmegov) [inline src/test/java/org/jacp/service/ImportServiceTest.java:37]: Minor: why static?

File: src/main/java/org/jacp/controller/ImportController.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    QuestionMapper questionMapper;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Map<String, String>> getAllImportsAndTest(@PathVariable Long id) {
+        QuestionEntity questionEntity = questionService.get(id);
+        QuestionTestFieldDto questionTestFieldDto = questionMapper.toTestFieldQuestionDto(questionEntity);
+        List<ImportEntity> importEntity = importService.getAll();
+        List<ImportDto> importDto = importMapper.toImportDto(importEntity);
+
+        Map<String, String> result = new HashMap<>();
```
[the remark this answers is not in the document]
Comment (saffchen) [reply src/main/java/org/jacp/controller/ImportController.java]: Fixed

File: src/main/java/org/jacp/controller/ImportController.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        String imports = importDto.stream()
```
[the remark this answers is not in the document]
Comment (saffchen) [reply src/main/java/org/jacp/controller/ImportController.java]: Fixed

File: src/test/java/org/jacp/service/ImportServiceTest.java (hunk 2 above)
Comment (saffchen) [reply src/test/java/org/jacp/service/ImportServiceTest.java:37]: this class has been removed

File: src/test/java/org/jacp/mapper/QuestionMapperTest.java (hunk 1 above)
Comment (saffchen) [reply src/test/java/org/jacp/mapper/QuestionMapperTest.java]: this class has been removed

File: src/main/java/org/jacp/controller/ImportController.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    @Autowired
+    ImportMapper importMapper;
+
+    @Autowired
+    QuestionMapper questionMapper;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Map<String, String>> getAllImportsAndTest(@PathVariable Long id) {
+        QuestionEntity questionEntity = questionService.get(id);
+        QuestionTestFieldDto questionTestFieldDto = questionMapper.toTestFieldQuestionDto(questionEntity);
```
[the remark this answers is not in the document]
Comment (saffchen) [reply src/main/java/org/jacp/controller/ImportController.java]: Fixed

**Review by pelmegov — APPROVED** [review]

**Review by freesky15 — APPROVED** [review]

**Pull request merged**